pkg/vz/network_darwin.go

```diff
@@ -37,7 +38,8 @@ func PassFDToUnix(unixSock string) (*os.File, error) {
 // DialQemu support connecting to QEMU supported network stack via unix socket.
 // Returns os.File, connected dgram connection to be used for vz.
 func DialQemu(unixSock string) (*os.File, error) {
-	unixConn, err := net.Dial("unix", unixSock)
+	network := "unix"
+	unixConn, err := net.DialUnix(network, nil, &net.UnixAddr{Name: unixSock, Net: network})
```

Comment:
Here again we don't get anything for making the code worse.

What if we add our own dialUnix() returning a *UnixCon, and hiding the ugly interface that standard library provides? We can use this helper in all 3 cases.

```go
func dialUnix(path string) (*UnixCon, error) {
    return net.DialUnix("unix", nil, &net.UnixAddr{Name: path, Net: "unix"})
}
```